installing_deps.sh

```diff
@@ -111,13 +111,13 @@ $AIL_HOME/doc/generate_modules_data_flow_graph.sh
 #### DB SETUP ####
 
 # init update version
-pushd ${AIL_HOME}
+#pushd ${AIL_HOME}
 # shallow clone
-git fetch --depth=500 --tags --prune
-git describe --abbrev=0 --tags | tr -d '\n' > ${AIL_HOME}/update/current_version
-echo "AIL current version:"
-git describe --abbrev=0 --tags
-popd
+#git fetch --depth=500 --tags --prune
+#git describe --abbrev=0 --tags | tr -d '\n' > ${AIL_HOME}/update/current_version
+#echo "AIL current version:"
+#git describe --abbrev=0 --tags
+#popd
```

Comment:
This part is needed to fetch the current tag. 
(I'll fix the travis build in the coming days)

Reply:
I commented this because the script couldn't recognize the git folder, have i done somethimg wrong?

Reply:
Will you fix this during the merge process?